apps/client/src/pages/CheckerPage.js

```diff
@@ -101,7 +107,7 @@ const CheckerPage = ({ checker, matomoTrackEvent, resetChecker, topic }) => {
 
     matomoTrackEvent({
       action: actions.CLICK_INTERNAL_NAVIGATION,
-      category: name.toLowerCase(),
```

Comment:
Can you undo this change? I'm refactoring `goToQuestion` and want to prevent merge conflicts. As discucced